sdk/loadtestservice/Azure.ResourceManager.LoadTestService/tests/ScenarioTests/LoadTestQuotaOperations.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.LoadTestService.Models;
+using Azure.ResourceManager.LoadTestService.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.LoadTestService.Tests.ScenarioTests
+{
+    public class LoadTestQuotaOperations : LoadTestServiceManagementTestBase
+    {
+        private QuotaResourceCollection _quotaResourceCollection { get; set; }
+        private QuotaResource _quotaResource { get; set; }
+        private QuotaResourceData _quotaResourceData { get; set; }
+
+        public LoadTestQuotaOperations(bool isAsync) : base(isAsync) //, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                await CreateCommonClient();
+            }
+            _quotaResourceCollection = Subscription.GetQuotaResources(LoadTestResourceHelper.RESOURCE_LOCATION);
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        [RecordedTest]
+        [AsyncOnly]
```

Comment:
Why do we add the `AsyncOnly` attribute in all the tests? We also require the methods to be tested in sync mode.

Reply:
Have removed this attribute